content/blog/2018/12/2018-12-10-outreachy-audit-log-plugin.adoc

```diff
@@ -0,0 +1,25 @@
+---
+:layout: post
+:title: Outreachy internships to add audit logging support to Jenkins
+:tags:
+- community
+- outreachy
+- outreachy2018
+:author: jvz
+---
+This year marks the first time the Jenkins project is participating in https://www.outreachy.org/[Outreachy].
+Outreachy is a program similar to Google Summer of Code (GSoC) where interns work on open source projects for a paid stipend.
+The key difference is that Outreachy reaches out to underrepresented groups and those who face systemic bias or discrimination in the technology industry in their home country.
+Once I learned about this program, I immediately volunteered to mentor as the concept strongly aligns with my ideals of inclusiveness and community building.
+I'm happy to report that both the Jenkins project,  and my employer [CloudBees](https://www.cloudbees.com), have been very supportive of this program.
```

Comment:
The two comments in this sentence are unnecessary.

I propose -> "I'm happy to report that both the Jenkins project and my employer [CloudBees](https://www.cloudbees.com) have been very supportive of this program."